src/main/proto/inter_node_rpc_service.proto

```diff
@@ -92,9 +95,12 @@ enum MetricEnum {
   QUEUE_CAPACITY = 7 [(additional_fields).name = "queue capacity", (additional_fields).description = "max capacity of the queue"];
   
   // cache
-  CACHE_EVICTION = 10 [(additional_fields).name = "cache eviction", (additional_fields).description = "cache eviction count"];
-  CACHE_HIT = 11 [(additional_fields).name = "cache hit", (additional_fields).description = "cache hit count"];
-  CACHE_MAX_SIZE = 12 [(additional_fields).name = "cache max size", (additional_fields).description = "max cache size in bytes"];
+  CACHE_EVICTION = 8 [(additional_fields).name = "cache eviction", (additional_fields).description = "cache eviction count"];
+  CACHE_HIT = 9 [(additional_fields).name = "cache hit", (additional_fields).description = "cache hit count"];
+  CACHE_MAX_SIZE = 10 [(additional_fields).name = "cache max size", (additional_fields).description = "max cache size in bytes"];
+
+  // Heap
+  HEAP_MAX = 11 [(additional_fields).name = "heap max", (additional_fields).description = "max heap size in bytes"];
```

Comment:
Let's leave some values for more cache metrics. We can start heap metrics from `16`.

Reply:
Using 16. Got my numbers wrong. Thanks